### ISSUE TITLE
Removal of unneccesary drawables

### ISSUE THREAD

**Issue by anubhakushwaha** [systers/malaria-app-android#74]
Copies of the same drawable has been maintained who just vary with respect to their sizes,color and opacity. These all can be implemented dynamically in their respective activities showing their current status via following methods invoked on the buttons after setting their background:

a)setAlpha(float value)                                        
b)setHighlightColor(int color)
c)setLayoutParams(LayoutParams param)

This highly reduces the number of copies made , for instance for all the buttons showing active and inactive state and many others like checked or not checked,etc.

**Comment by shamathmika** [comment]
@anubhakushwaha 
Can I work on this?

**Comment by anubhakushwaha** [comment]
@shamathmika it's the mentor who gives the permission to go ahead and work on an issue especially when it's provided by some contributor other than the mentor,it's not yet known if this issue has been resolved by the assigned gsocer this year or even if it's of prime importance for the project as a whole,a person other than the mentors cannot  give you any permission to work ahead or not until the issue itself being approved.The same applies for all the open issues not approved the the mentors.I hope I was clear.Better you pick one which has been approved the mentor and it's PR not yet submitted or make a new issue.

**Comment by shamathmika** [comment]
@anubhakushwaha 
Okay, thank you.

**Comment by anubhakushwaha** [comment]
@MeepyMay can I go ahead with this because since long no mentor has been responding to the open issues. thanks

**Comment by yatri1609** [comment]
@geekanamika @m-murad @ExactlyMay Can I go ahead with this? Since it is open since long time.

**Comment by geekanamika** [comment]
@yatri1609  Sure, Go ahead :) Fetch code from `gsoc18-code` branch as it contains the latest updates.

**Comment by yatri1609** [comment]
@geekanamika Sure thanks

**Comment by geekanamika** [comment]
Resolved through #353  , closing this issue.
